.github/actions/create-github-release-action/README.md

```diff
@@ -0,0 +1,51 @@
+<!--
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: 2025 The Linux Foundation
+-->
+
+# ⬆️ Create Github Release Action
+
+This action creates a new release in the calling repo using the provided version.
+
+## create-github-release-action
+
+## Usage Example
```

Comment:
Are these 2 headers in the correct level? I am asking this because, as-is, it may seem that the `create-github-release-action` has missing content its section, no? Or should they be `##` and `###` for `create-github-release-action` and `Usage Example`, respectively?

Reply:
Yeah, the action title is unnecessary. Removed.